src/routes/github/create-branch/github-branches-get.ts

```diff
@@ -17,8 +17,15 @@ export const GithubBranchesGet = async (req: Request, res: Response): Promise<vo
 
 	try {
 		const gitHubUserClient = await createUserClient(githubToken, jiraHost, req.log, gitHubAppConfig.gitHubAppId);
-		const branches = await gitHubUserClient.getReferences(owner, repo, 100);
-		res.send(branches);
+		const [ branches, repository ] = await Promise.all([
+			gitHubUserClient.getReferences(owner, repo),
```

Comment:
Is it returning all branches? if not, then make sure default branch is part of the response.

Reply:
Thats actually a good point.This request only returns the first 100 branches. 
So updated the response of this endpoint to return:
``````
{  
    branches: [] , // all the branches filtering out the default branch 
    defaultBranch: XXXX
}
``````